README.md

```diff
@@ -168,6 +168,14 @@ By default, `snyk-monitor` uses an emptyDir for temporary storage. If you prefer
 The PVC's name defaults to `snyk-monitor-pvc`. If you prefer to override this, then use the following value:
 * `pvc.name`
 
+## Using custom CA certificate
+
+You can use a custom CA certificate by setting the following variable in the `ConfigMap`:
```

Comment:
I think I need to add a sentence about _why_ someone would add a custom CA cert? (not sure what to write)